.github/workflows/build.yml

```
@@ -0,0 +1,162 @@
+          {
+            echo "commit_message<<${delimiter}"
+            echo "$COMMIT_MESSAGE"
+            echo "${delimiter}"
+          } >> $GITHUB_OUTPUT
+          echo "branch_name=$BRANCH_NAME" >> $GITHUB_OUTPUT
+          echo "target_branch_name=$TARGET_BRANCH_NAME" >> $GITHUB_OUTPUT
+          echo "previous_commit=$PREVIOUS_COMMIT" >> $GITHUB_OUTPUT
+
+      - id: verify_out
+        name: Write verify job matrix
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2012:info:1:17: Use find instead of ls to better handle non-alphanumeric filenames

(shellcheck)